workflows/mhcquant.nf

```diff
@@ -314,49 +320,38 @@ workflow MHCQUANT {
     }
 
     OPENMS_MZTABEXPORTER(ch_output)
-    ch_versions = ch_versions.mix(OPENMS_MZTABEXPORTER.out.versions.ifEmpty(null))
+    ch_versions = ch_versions.mix(OPENMS_MZTABEXPORTER.out.versions)
 
     //
     // SUBWORKFLOW: Predict class I (neoepitopes)
     //
-    if (params.predict_class_1 & !params.skip_quantification) {
-        PREDICT_CLASS1 (
-            OPENMS_MZTABEXPORTER.out.mztab,
-            peptides_class_1_alleles,
-            ch_vcf_from_sheet
-        )
-        ch_versions = ch_versions.mix(PREDICT_CLASS1.out.versions.ifEmpty(null))
-        ch_predicted_possible_neoepitopes = PREDICT_CLASS1.out.ch_predicted_possible_neoepitopes
-    } else {
-        ch_predicted_possible_neoepitopes = Channel.empty()
-    }
-
-    //
-    // SUBWORKFLOW: Predict class II (neoepitopes)
+    // TODO: Temporary disabled because of outdated vcf parsing
+    //if (params.predict_class_1 & !params.skip_quantification) {
+    //    PREDICT_CLASS1 (
+    //        OPENMS_MZTABEXPORTER.out.mztab,
+    //        peptides_class_1_alleles,
+    //        ch_vcf_from_sheet
+    //    )
+    //    ch_versions = ch_versions.mix(PREDICT_CLASS1.out.versions)
+    //    ch_predicted_possible_neoepitopes = PREDICT_CLASS1.out.ch_predicted_possible_neoepitopes
+    //} else {
+    //    ch_predicted_possible_neoepitopes = Channel.empty()
+    //}
```

Comment:
maybe link an issue about this for future devs?

Reply:
Documented here: https://github.com/nf-core/mhcquant/issues/248
Unfortunately not an easy fix 😁 but I hope to use some module from epitopeprediction in the future to fix this

Reply:
Maybe alter the README since this is not supported ATM